### PR TITLE
UI scaling

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Adds scaling functionality, which attempts to scale the wallet window according to the size of the screen.

Doesn't seem to be completely consistent on a multi-monitor setup when the wallet is opened by clicking the icon (the screen registered is always the primary display, even though the chrome window is on a secondary display). Might be a MacOS only thing.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: packages/browser-wallet/src/popup/styles/config/_functions.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@use 'sass:math';
+
+@function strip-unit($value) {
+    @return math.div($value, ($value * 0 + 1));
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/styles/config/_functions.scss:4]: what is x * 0? :thinking: 
I'm confused by this

**Comment by shjortConcordium** [comment]
I don't think I have a problem with the multi-monitors, maybe because they have the same size, though.
It does seem that the text have smaller, even though the size of the window is bigger.

File: packages/browser-wallet/src/popup/styles/config/_functions.scss (hunk 1 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/styles/config/_functions.scss:4]: It's a commonly known hackish solution to strip any number value from it's units, f.x. `strip-unit(18px) -> 18`.

**Comment by soerenbf** [comment]
> I don't think I have a problem with the multi-monitors, maybe because they have the same size, though. It does seem that the text have become smaller, even though the size of the window is bigger.

Yes, it's a problem when running monitors of different sizes.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**